src/app/features/search/single-search-result/single-search-result.component.ts

```diff
@@ -20,17 +20,20 @@ export class SingleSearchResultComponent implements OnInit {
   constructor(private technicalRecordService: TechnicalRecordService) {}
 
   ngOnInit(): void {
-    this.technicalRecordService.viewableTechRecord$(this.vehicleTechRecord, this.ngDestroy$).subscribe(
-      record =>
-        (this.vehicleDisplayData = {
-          vin: this.vehicleTechRecord.vin,
-          vrm: this.vehicleTechRecord.vrms.find(vrm => vrm.isPrimary)?.vrm,
-          make: record?.chassisMake,
-          model: record?.chassisModel,
-          manufactureYear: record?.manufactureYear,
-          vehicleType: record?.vehicleType.toUpperCase()
-        })
-    );
+    this.technicalRecordService
+      .viewableTechRecord$(this.vehicleTechRecord)
+      .pipe(takeUntil(this.ngDestroy$))
```

Comment:
Component does not implement on destroy, take(1) ?

Reply:
implemented ngOnDestroy